### ISSUE TITLE
InexactError when reading a CSV file

### ISSUE THREAD

**Issue by KuiPN** [JuliaData/CSV.jl#397]
I have a test.csv file that looks like this:
```
COL1,COl2,COL3
123,ABC,9398.193419185
456,DEF,74810199.033988851037472901827191090834
```
I used a simple code to read in the file:
```
CSV.File("test.csv") |> DataFrame
```
But encountered the following error:
```
ERROR: InexactError: check_top_bit(Int64, -3)
```
It appears that the problem is in the 2nd row that has too high a precision. If I remove, say, the last 5 digits of that large number, the code will read the file correctly.

**Comment by KuiPN** [comment]
by the way, this is my environment:
```
(v1.1) pkg> status CSV
    Status `~/.julia/environments/v1.1/Project.toml`
  [336ed68f] CSV v0.4.3
  [a93c6f00] DataFrames v0.17.1
```

**Comment by quinnj** [comment]
This is solved in #395 by updating our Parsers.jl requirement.
